src/tailwind.config.ts

```diff
@@ -120,11 +120,7 @@ const plugins = [
 ]
 
 export default {
-  content: [
-    './src/**/*.{vue,js,ts}',
-    './demo/index.html',
-    './demo/**/*.vue',
-  ],
+  content: ['./src/**/*.{vue}'],
```

Comment:
We're already doing this in the top-level block so this one probably just needs the vue src glob?